src/virtual_elements.js

```diff
@@ -87,30 +87,38 @@ const elementOpen = function(tag, key, statics, var_args) {
    */
   const attrsArr = data.attrsArr;
   const newAttrs = data.newAttrs;
-  let attrsChanged = false;
+  const isNew = !attrsArr.length;
```

Comment:
We can definitely do that. Another PR, though, because we can do the same for `keyMap`.